src/BizHawk.Emulation.Common/Base Implementations/Bk2MnemonicLookup.cs

```diff
@@ -904,7 +904,7 @@ public static string LookupAxis(string button, string systemId)
 				["RightBracket"] = ']',
 				["Semicolon"] = ';',
 				["Quote"] = '\'',
-				["Period"] = '.',
+				["Period"] = 'p', // because '.' represents no input
```

Comment:
We should have a unit test for this.

Reply:
Agreed, I most certainly did it too.

Reply:
I added an assert instead https://github.com/TASEmulators/BizHawk/commit/a41817b7030fec528aff5da57b88e543a8c5ea68